### ISSUE TITLE
UI Pages Refactoring

### ISSUE THREAD

**Issue by Confectrician** [openhab/openhab-docs#1968]
Now that UI add-ons docs are available some documentation could be further deduplicated:

E.g. replace:

* https://www.openhab.org/docs/ui/habot/#habot  
  with  
  https://www.openhab.org/addons/ui/habot/

* https://www.openhab.org/docs/configuration/habpanel.html  
  with  
  https://www.openhab.org/addons/ui/habpanel/doc/habpanel.html (needs some fixing to get the images loaded)

_Originally posted by @wborn in https://github.com/openhab/openhab-docs/issues/1935#issuecomment-1357400059_

**Comment by Confectrician** [comment]
We should remove the specific ui pages here and merge/sync them with the addon readmes before doing so.
Also we should then temporary add `301` redirects from docs to addons via website repository, to inform google about those changes.

We should be able to leave the sidebar elements for a while and redirect directly to the adons pages.

**Comment by wborn** [comment]
Yes that would be nice! There's also some [link replacement logic in prepare-docs.rb](https://github.com/openhab/website/blob/main/prepare-docs.rb#L229) that may need an update for this.
